source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/details/index.tsx

```diff
@@ -1,37 +1,55 @@
 import * as React from 'react';
-
-import { translate, InjectedTranslateProps as i18nProps } from 'react-i18next';
 import { compose } from 'recompose';
-import { ProjectAttributes } from '@data/models/project';
-import { ResourceObject } from 'jsonapi-typescript';
-import { PROJECTS_TYPE } from '@data';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import { ProjectResource, ApplicationTypeResource, attributesFor } from '@data';
+import { withTranslations, i18nProps } from '@lib/i18n';
+
 
 interface Params {
-  project: ResourceObject<PROJECTS_TYPE, ProjectAttributes>;
+  project: ProjectResource;
+  applicationType: ApplicationTypeResource;
 }
 
 type IProps =
   & Params
   & i18nProps;
 
+const mapRecordsToProps = (passedProps) => {
+
+  const { project } = passedProps;
+
+  return {
+    applicationType: q => q.findRelatedRecord(project, 'type')
+  };
+};
+
 class Details extends React.Component<IProps> {
 
   render() {
 
-    const { t, project } = this.props;
-    const { language, type, description } = project.attributes;
+    const { t, project, applicationType } = this.props;
+    const { language, description } = attributesFor(project);
+    const { description: type } = attributesFor(applicationType);
 
     return (
-      <div className='details'>
+      <div data-test-project-details className='details'>
         <h3>{t('project.details.title')}</h3>
         <div className='flex justify-content-space-around'>
           <div className='flex-grow'>
             <h4>{t('project.details.language')}</h4>
-            <p style={{marginRight: '39px'}}>{language}</p>
+            <p
+              data-test-project-detail-language
+              style={{marginRight: '39px'}}
```

Comment:
I fixed that but it's not part of this PR, I just added the data-test-project-detail-language in case we need later on. I can do the styling change though